app/adapters/alma/bib.rb

```diff
@@ -13,15 +14,25 @@ class Bib
     # @param records an array of MARC::Record records
     class << self
       def get_bib_record(ids, conn=nil, opts={})
-        res = Alma::Connector.connection.get "bibs?mms_id=#{self.ids_remove_spaces(ids: ids)}", {
-          :apikey => self.apikey, # I can't stub this in the bib_spec
-          :expand => "p_avail,e_avail,d_avail,requests",
-          :view => "full"
-        }
-        reader = MARC::XMLReader.new(StringIO.new(res.body))
-        return reader.first unless self.ids_build_array(ids: ids).count > 1
-        records = []
-        reader.select {|record| records << record}
+        res = Alma::Connector.connection.get("bibs?mms_id=#{ids}",
+        {query: { :expand => "p_avail,e_avail,d_avail,requests" }, :apikey => self.apikey},
+        {'Accept' => 'application/xml'} )
+
+        doc = Nokogiri::XML(res.body)
+        doc_unsuppressed(doc)
+        unsuppressed_marc.first
+      end
+
+      def get_bib_records(ids, conn=nil, opts={})
```

Comment:
Can we add yard docs here, too?

Reply:
It might be nice if this method would accept an array of IDs and turn them into the string required by the API.

Reply:
Thank you @hackmastera for editing with me the yard docs!